frontend/src/components/app/index.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Home from '../home';
+
+const App = () => {
+  return <Home />;
+};
```

Comment:
nitpicky, but this can written more succinctly as
`const App = () => (<Home />)`

Reply:
Good spot, I feel there might be a liniting option for this